packages/schematics/src/migrations/update-3-0-0/update-3-0-0.ts

```diff
@@ -118,6 +118,72 @@ function applyRecommendedExtraExtends(host: Tree) {
   return chain(rules);
 }
 
+function removeNegativeValuesFromComponentMaxInlineDeclarations(
+  rule: Linter.RuleEntry | undefined,
+) {
+  if (!Array.isArray(rule) || rule.length !== 2) return;
+  const [, currentSchema] = rule;
+  rule[1] = Object.entries(currentSchema).reduce(
+    (accumulator, [key, value]) => {
+      return Number(value) < 0 ? accumulator : { ...accumulator, [key]: value };
+    },
+    {},
+  );
+}
+
+function updateComponentMaxInlineDeclarationsSchema({
+  overrides,
+  rules,
+}: Linter.Config) {
+  removeNegativeValuesFromComponentMaxInlineDeclarations(
+    rules?.['component-max-inline-declarations'],
```

Comment:
Do we need to specify it as `"@angular-eslint/component-max-inline-declarations"` or it just work as is? I just saw that in v2 migration, you used `use-pipe-decorator`...